### DISCUSSION TITLE
Fold Change Calculation Related Issues

### DISCUSSION THREAD

**Discussion by luosupercool** [Nesvilab/FragPipe-Analyst-doc#65] [Q&A] [answered]
Dear fragpipe-analyst developers,

I have questions about the fold change in the DE_results.csv file obtained after the software does or does not use the VSN or median centering method.  

I would like to understand the correlation between the values in the Imputed_matrix.csv file and the fold change. I am working with DIA data, and I wonder if the fold change (FC) is calculated using the raw quantification values from the report.pg_matrix.tsv file output by DIANN when no normalization method is specified, as I found that this corresponds with the DE_results.csv file. However, I have other questions, such as why different normalization methods result in different FCs in the DE_results. Could you please tell me the general data processing steps of fragpipe-analyst? Thank you very much!

ctuo

**Comment by hsiaoyi0504** [comment, marked as answer]
Duplicated as https://github.com/Nesvilab/FragPipe-Analyst/issues/66.
